src/pages/SubmissionConfig.js

```diff
@@ -360,7 +366,22 @@ export const SubmissionConfig = ({ methodId }) => {
           ]),
           div({ style: { lineHeight: 2.0, marginTop: '1.5rem' } }, [
             div([h(TextCell, ['This will launch ', span({ style: { fontWeight: 'bold' } }, [_.keys(selectedRecords).length]), ' workflow(s).'])]),
-            h(TextCell, { style: { marginTop: '1rem' } }, ['Running workflows will generate cloud compute charges.'])
+            h(TextCell, { style: { marginTop: '1rem' } }, ['Running workflows will generate cloud compute charges.']),
+            ...Utils.cond([workflowSubmissionError, () => [
+              div({ style: { display: 'flex', alignItems: 'center', marginTop: '1rem' } }, [
+                icon('warning-standard', { size: 16, style: { color: colors.danger() } }),
+                h(TextCell, { style: { marginLeft: '0.5rem' } }, ['Error submitting workflow:'])
+              ]),
+              div({
+                style: {
+                  padding: '0.5rem', backgroundColor: colors.light(),
+                  whiteSpace: 'pre-wrap', overflowY: 'scroll', overflowWrap: 'break-word',
+                  fontFamily: 'Menlo, monospace',
```

Comment:
I think we should let the font be set to whatever is default. Any reasons to set it to this?

Reply:
This font/styling was picked to be similar to the normal error popup styling [here](https://github.com/DataBiosphere/cbas-ui/blob/62669cf1aad4bd869e32e89b2954b71b0a412d1f/src/components/ErrorView.js#L15-L20C4). To keep it synced up I can export the style from ErrorView and import it here if that would be better/clearer?